plugins/hydra_ray_launcher/integration_test_tools/setup_integration_test_ami.py

```diff
@@ -3,7 +3,9 @@
 from datetime import datetime
 
 dependencies = [
-    "ray>=1.0.0",
+    "ray==1.1.0",
+    "cloudpickle==1.6.0",
+    "pickle5==0.0.11",
```

Comment:
can you explain why you need to add the two new dependencies?

Reply:
The dependencies are always needed - basically we need to mirror whatever is in `setup.py` for hydra-ray-launcher.

I thought about moving all these requirements to a separate `requriements.txt` file so both `setup.py` and this script could share, but that does not work with how `nox` is run - nox is run from root folder when installing ray launcher and as a result the `requirements.txt` file cannot be located correctly.

I don't think it is worth it to change the whole nox file only to make this work, what do you think?

Reply:
Okay. not worth this to leak into the already complicated noxfile for sure.